SQL.test.js

```diff
@@ -498,6 +498,50 @@ test('should be able to use nested SQLStatements in template literal', t => {
   t.end()
 })
 
+test('should be able to use the result of SQL.glue([SQL``, SQL``], separator) result with multiple values inside the first element of SQL.glue', t => {
+  const ids = [1, 2, 3]
+  const name = 'foo'
+  const inIds = SQL.glue(
+    ids.map((id) => SQL`${id}`),
+    ' , '
+  )
+  const condition = SQL`tsd.id IN (${inIds})`
+  // this condition results in:
+  // {
+  //     "strings": [
+  //         "tsd.id IN (",
+  //         ")"
+  //     ],
+  //     "_values": [
+  //         {
+  //             "strings": [
+  //                 "",
+  //                 " , ",
+  //                 " , ",
+  //                 " "
+  //             ],
+  //             "_values": [
+  //                 1,
+  //                 2,
+  //                 3
+  //             ]
+  //         }
+  //     ]
+  // }
```

Comment:
do we need this long comment here?

Reply:
I left this comment to clarify the reason for the bug, but I removed it and left it only in the PR description